Userland/Libraries/LibWeb/HTML/MediaError.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2023, Tim Flynn <trflynn89@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/String.h>
+#include <LibJS/Forward.h>
+#include <LibWeb/Bindings/PlatformObject.h>
+
+namespace Web::HTML {
+
+class MediaError final : public Bindings::PlatformObject {
+    WEB_PLATFORM_OBJECT(MediaError, Bindings::PlatformObject);
+
+public:
+    enum class Code : u16 {
+        Aborted = 1,
+        Network,
+        Decode,
+        SrcNotSupported,
```

Comment:
Since these have specified integer values, I think it would be nice to specify them here explicitly (even if they are just 1, 2, 3, 4 :)

Reply:
Sure!